python/lint/flake8.ini

```diff
@@ -0,0 +1,5 @@
+[flake8]
+max-line-length = 100
+max-complexity = 10
+exclude = *build*,*artifacts*
+ignore = D100,D101,D102,D103,D104,D105,D106,D107,D203,D213,W504
```

Comment:
not sure about that list - it's some arbitrary list of Reporting's ancestors, right?

Reply:
Right. But this list is reasonable.